taskRequests/script.js

```diff
@@ -84,6 +93,173 @@ function openTaskDetails(id) {
   window.location.href = url;
 }
 
+if (isDev) {
+  containerFilter.style.display = 'flex';
+  oldContainerFilter.style.display = 'none';
+} else {
+  containerFilter.style.display = 'none';
+  oldContainerFilter.style.display = 'flex';
+}
+sortButton.addEventListener('click', async (event) => {
+  event.stopPropagation();
```

Comment:
Re: is this required? line [121] (https://github.com/Real-Dev-Squad/website-dashboard/pull/576/files#diff-23b61ff1789eab0a1640feae99a0bf3465052f6c0cdda786aa4ae08c876715b5R121)

Reply:
thanks for reminding got it removed